### ISSUE TITLE
Cannot sync Wiimote model PT BR 04FC under Windows 10 v1709

### ISSUE THREAD

**Issue by Expack3** [jloehr/HID-Wiimote#36]
I'm running the Windows 10 Fall Creators Update (v1709) on my laptop, and while the driver appears to install correctly, and I can get the Wiimote added to my list of devices using the "Join a Personal Area Network" solution, the controller settings show nothing, and attempting to update the driver claims I have the latest version.

My laptop's Bluetooth chipset is Intel Wireless Bluetooth, with driver version 20.0.0.11.

**Comment by jloehr** [comment]
Is it a genuine Wii Remote or some third party? Can you check its hardware ID and the actual loaded driver for it via device manager (make sure to look at the HID device node and not the Bluetooth device node).

**Comment by Expack3** [comment]
It’s the Wiimote which came with the Wii I bought it when the Wii was originally released – I’m pretty confident it’s the genuine device.

 

Here’s the HID (assuming that means “Hardware ID”) I got:  BTHENUM\Dev_00191D3FC858

**Comment by jloehr** [comment]
That's the wrong Hardware ID. Check under Human Interface Device for the HID device node. It'll be either labeled as "Bluetooth HID Device" or "Wiimote Device". In case of the former, check the Events tab and try to manually change the driver. In case of the latter or if there is no HID device node, make sure the Wii Remote is connected and doesn't turn off.

I think someone else mentioned he had an issue with the Fall Creators Update as well. I haven't upgraded yet due to some other project.

**Comment by Mohammed-JS-Khan** [comment]
Not sure if this helps at all:

http://touchmote.net/
> NOTICE New Wiimotes (S/N ending with Z-C4, Z-C6 or C-C4) will not work with Microsoft Bluetooth Stack. We recommend you to get a Wiimote manufactured before November 2011. To get the newer Wiimotes to work you have to use the Mayflash DolphinBar or similar, alternatively you might try this guide to install the Toshiba Bluetooth Stack.

I spent like 30s last night looking for the S/N but I decided against it, and just used one of my older controllers, which worked without a flaw.

I wonder if the Toshiba Bluetooth Stack would help this project out.

**Comment by jloehr** [comment]
Nope, that issue is unrelated. In fact my driver replaces the Windows driver that causes TR-Wiimotes to not work.
